pkg/converter/convert.go

```diff
@@ -43,6 +43,30 @@ const (
 	execAPIVersion  = "client.authentication.k8s.io/v1beta1"
 )
 
+func getServerId(authInfoPtr *api.AuthInfo) (serverId string) {
+	if authInfoPtr == nil || authInfoPtr.Exec == nil || authInfoPtr.Exec.Args == nil {
+		return
+	}
+	if len(authInfoPtr.Exec.Args) < 1 {
+		return
+	}
+	for i := range authInfoPtr.Exec.Args {
+		if authInfoPtr.Exec.Args[i] == "--server-id" {
+			if len(authInfoPtr.Exec.Args) > i+1 {
+				return authInfoPtr.Exec.Args[i+1]
+			}
+		}
+	}
+	return
+}
+
+func isUsingKubelogin(execConfigPtr *api.ExecConfig) (ok bool) {
+	if execConfigPtr == nil {
+		return
+	}
+	return execConfigPtr.Command == "kubelogin"
```

Comment:
i think we should check if Command contains kubelogin as it may be ./kubelogin, /path/to/kubelogin, or even kubelogin.exe